src/cmd_line/subparsers/file.ts

```diff
@@ -9,7 +9,7 @@ export function parseEditFileCommandArgs(args: string): node.FileCommand {
     }
 
     var scanner = new Scanner(args);
-    let name = scanner.nextWord();
+    var name = scanner.nextWord();
```

Comment:
Wait, why did you change _more_ `let`s to `var`s? That's the opposite of what I want! 😉